README.md

```diff
@@ -20,14 +24,20 @@ Just raise [an issue](https://github.com/COSIMA/cosima-recipes/issues) explainin
 
 ## Contents
 
-### [Tutorials](https://cosima-recipes.readthedocs.io/en/latest/tutorials.html)
+We are in the process of transition these recipes from using [cosima-cookbook](https://github.com/COSIMA/cosima-cookbook) infrastructure to an _intake catalogue_, so you will find recipes may use either method to access model data. If you are halfway through a project using the cosima-cookbook and looking for resources, this [tag](https://github.com/COSIMA/cosima-recipes/tree/cosima_cookbook) marks the repository before most recipes were transitioned.
 
-The notebook `ACCESS-NRI_Intake_Catalog.ipynb` outlines the basic philosophy of the Intake catalog. This is the best place to start if you are not familiar with the Intake catalog. Also included here are some other tutorials, either related to the cookbook (e.g., `Make_Your_Own_Database.ipynb`) or more general (`Making_Maps_with_Cartopy.ipynb`).
+### [Tutorials](https://cosima-recipes.readthedocs.io/en/latest/tutorials.html)
 
-Don't miss out the <a href="https://nbviewer.jupyter.org/github/COSIMA/cosima-recipes/blob/master/Tutorials/Using_Explorer_tools.ipynb" target="_blank">tutorial</a> about using `cosima-cookbook`'s `explore` submodule to find out about available experiments and variables in a database. (The `explorer` tutorial is better viewed either via nbviewer or by running the jupyter notebook yourself.)
+The notebook [ACCESS-NRI_Intake_Catalog](https://cosima-recipes.readthedocs.io/en/latest/Tutorials/ACCESS-NRI_Intake_Catalog.html) outlines the basic philosophy of the Intake catalog and how to transition from using the cosima-cookbook to the Intake catalogue. This is the best place to start if you are not familiar with the Intake catalog. Also included here are some other tutorials, related to techniques (e.g. [Making_Maps_with_Cartopy.ipynb](https://cosima-recipes.readthedocs.io/en/latest/Tutorials/Making_Maps_with_Cartopy.html)) or tools (e.g. [Model Agnostic Analysis](https://cosima-recipes.readthedocs.io/en/latest/Tutorials/Model_Agnostic_Analysis.html))
```

Comment:
Let's avoid the "transition"? Let's just point people to the Intake. Let's detach from the legacy of the past?

Reply:
The tutoral goes into a lot of detail about transitioning. Its the content of the tutorial !

Reply:
OK. Perhaps I'm suggesting we need a new tutorial detached from the past and looking to the future.

But that's a different PR.